lib/src/native/realm_core.dart

```diff
@@ -152,6 +152,16 @@ class _RealmCore {
     _realmLib.realm_config_set_schema_version(config.handle._pointer, version);
   }
 
+  bool getReadOnlyMode(Configuration config) {
```

Comment:
```suggestion
  bool getConfigReadOnly(Configuration config) {
```

Reply:
since we are going to have a bunch of these flags in the config I suggest we prefix the methods with `Config` or `Configuration`